Snakefile

```diff
@@ -4,24 +4,22 @@ import glob
 
 # obtaining plate_ids
 sql_paths = glob.glob("./data/*.sqlite")
-PLATE_IDS = [Path(sql_file).stem for sql_file in sql_paths]
+PLATE_IDS = glob_wildcards("data/{id}.sqlite").id
 
 
 # importing DAGs
 include: "rules/preprocessing.smk"
-
-
 include: "rules/feature_select.smk"
 
 
 rule all:
     input:
-        expand("results/preprocessing/{plate_id}_aggregate.csv.gz", plate_id=PLATE_IDS), # expected outputs from the first DAG "Preprocessing"
+        expand("results/preprocessing/{plate_id}_aggregate.csv.gz", plate_id=PLATE_IDS),  # expected outputs from the first DAG "Preprocessing"
         expand("results/preprocessing/{plate_id}_cell_counts.tsv", plate_id=PLATE_IDS),
         expand("results/preprocessing/{plate_id}_augmented.csv.gz", plate_id=PLATE_IDS),
         expand("results/preprocessing/{plate_id}_normalized.csv.gz", plate_id=PLATE_IDS),
         expand(
             "results/preprocessing/{plate_id}_feature_select.csv.gz",
             plate_id=PLATE_IDS,
         ),
-        "results/preprocessing/consensus.csv.gz",
+        "results/preprocessing/median_consensus.tsv.gz",
```

Comment:
still need to fix this (I think how you had it before was better?)

e.g. what if the user performed `modz` consensus?

Reply:
Before it was `consensus`, a more general name. I can revert it back. Still figuring out how to switch the output name dynamically, since `snakemake` is very `strict` about the names of output files!